astra/src/main/java/com/slack/astra/chunk/ReadWriteChunk.java

```diff
@@ -243,14 +245,17 @@ public boolean snapshotToS3(String bucket, String prefix, BlobFs blobFs) {
       // Upload files
       logger.info("{} active files in {} in index", filesToUpload.size(), dirPath);
       for (String fileName : filesToUpload) {
-        logger.debug("File name is {}}", fileName);
+        File fileToCopy = new File(dirPath.toString(), fileName);
```

Comment:
You may want to use something like `FileUtils.size()` here, or `Files.size()` so that you don't have to create a new file object just to get the bytes.